src/unittest/test_servermodmanager.cpp

```diff
@@ -48,22 +51,25 @@ static TestServerModManager g_test_instance;
 
 void TestServerModManager::runTests(IGameDef *gamedef)
 {
-	const char *saved_env_mt_subgame_path = getenv("MINETEST_SUBGAME_PATH");
-	const char *saved_env_mt_mod_path = getenv("MINETEST_MOD_PATH");
-#ifdef WIN32
-	{
-		std::string subgame_path("MINETEST_SUBGAME_PATH=");
-		subgame_path.append(TEST_SUBGAME_PATH);
-		_putenv(subgame_path.c_str());
+	if (!findSubgame(SUBGAME_ID).isValid()) {
+		warningstream << "Can't find game " SUBGAME_ID ", skipping this module." << std::endl;
+		return;
+	}
 
-		std::string mod_path("MINETEST_MOD_PATH=");
-		mod_path.append(TEST_MOD_PATH);
-		_putenv(mod_path.c_str());
+	auto test_mods = getTestTempDirectory().append(DIR_DELIM "test_mods");
+	{
+		auto p = test_mods + (DIR_DELIM "test_mod" DIR_DELIM);
+		fs::CreateAllDirs(p);
+		std::ofstream ofs1(p + "mod.conf", std::ios::out | std::ios::binary);
+		ofs1 << "name = test_mod\n" << "description = Does nothing\n";
```

Comment:
```suggestion
		ofs1 << "name = test_mod\n"
				"description = Does nothing\n";
```
... or similar. New lines should be put on new lines for readability.